packs/equipment-effects/effect-diplomats-badge.json

```diff
@@ -22,6 +22,7 @@
         },
         "rules": [
             {
+                "label": "Diplomat's Bearing",
```

Comment:
This label should be localized. Consider using a unique slug in addition to / instead of the label.